packages/contracts-bedrock/test/L1/SharedLockbox.t.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+import { Unauthorized } from "src/libraries/errors/CommonErrors.sol";
+
+// Targent contract
+import { SharedLockbox } from "src/L1/SharedLockbox.sol";
+
+// Interfaces
+import { IOptimismPortal } from "src/L1/interfaces/IOptimismPortal.sol";
+
+contract SharedLockboxTest is Test {
```

Comment:
What do u think about using `CommonTest` and creating the lockbox in the `Setup.sol` like other L1 contracts?

So it is consisten and also available for other tests

Reply:
All L1 contracts do that so I believe we should use it too so it remains consistent

In a future migraton they can update all contracts together

Reply:
All the other L1 contracts use `0.8.15`. Should I use it as well?

Reply:
ah erfect